### PR TITLE
/logs docs fixes: Attributes section

### PR DESCRIPTION
**Pull request by ryanoblak-okta**
## Description:
This changeset fixes a few errors in the "Attributes" section of our
`/logs` documentation:
  - In the table, `eventId` should be `uuid`.
  - There are references to `requestId` that should be `transaction.id`.
  - There are references to `sessionId` that should be
    `authenticationContext.externalSessionId`.

Also, in the "Example Log object" section, the reference to
`authentication_context` should be `authenticationContext`.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-127977](https://oktainc.atlassian.net/browse/OKTA-127977)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [n/a] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [n/a] For functional changes, I have added tests.

## Primary Reviewers:

* @mystiberry-okta 
* @bobtiernay-okta 

- *Tag reviewer(s)*
* @aminjashki-okta
* @rpamidimarri-okta
* @ericzhou-okta
* @richardblaylock-okta
* @jeffpeeler-okta
* @guyzehavi-okta
* @chrismckerracher-okta

### CODE REVIEW

File: _source/_docs/api/resources/system_log.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ Log objects are read-only. The following properties are available:
 |-----------+-----------------------------------------------------------------------+----------------------------------------------------------------+----------+--------+----------+-----------+-----------|
 | Property  | Description                                                           | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
 | -------   | --------------------------------------------------------------------- | ---------------------------------------------------------------| -------- | ------ | -------- | --------- | --------- |
-| eventId   | Unique key for an event                                               | String                                                         | FALSE    | TRUE   | TRUE     |           |           |
+| uuid      | Unique identifier for an indvidiual event                             | String                                                         | FALSE    | TRUE   | TRUE     |           |           |
```
Comment (bobtiernay-okta) [inline _source/_docs/api/resources/system_log.md]: `indvidiual` -> sp

**Comment by mystiberry-okta** [comment]
@ryanoblak-okta please pardon my ignorance, but it looks like you modified this assuming someone is using an SDK? These docs assume direct API and don't usually mention methods supplied in the SDK. Not sure I understand this correctly, so push back if this sounds wrong.

File: _source/_docs/api/resources/system_log.md [hunk 2]
```diff
@@ -324,7 +324,7 @@ Log objects are read-only. The following properties are available:
 
 > The actor and/or target of an event is dependent on the action performed. All events have actors. Not all events have targets.
 
-> The `sessionId` can identify multiple requests.  A single `requestId` can identify multiple events.  Use the `sessionId` to link events and requests that occurred in the same session.
+> The `authenticationContext.externalSessionId` can identify multiple requests.  A single `transaction.id` can identify multiple events.  Use the `authenticationContext.externalSessionId` to link events and requests that occurred in the same session.
```
Comment (bobtiernay-okta) [inline _source/_docs/api/resources/system_log.md]: Do we need a line to say "Use the `transaction.id` to link events that occurred in the same request", perhaps rephrasing the last sentence to "Use the `authenticationContext.externalSessionId` to link requests that occurred in the same session" as that would be implied by the first.

**Comment by ryanoblak-okta** [comment]
Hi @mystiberry-okta 
> please pardon my ignorance, but it looks like you modified this assuming someone is using an SDK? These docs assume direct API and don't usually mention methods supplied in the SDK. Not sure I understand this correctly, so push back if this sounds wrong.

These changes apply to the direct API, not any of our SDKs. The dot notation I'm using (e.g. `authenticationContext.externalSessionId`) is a shorthand for referring to parts of the structured JSON response.

**Pull request merged**